README.md

```diff
@@ -1,50 +1,56 @@
 # Music Player - Python
 
-A `Music Player` application that allows you to listen to music files stored in the computer's internal or external memory.
+A `Music Player` application that allows you to listen to musics that stored in the computer's internal or external memory..
 
-![Table Image](images/sc1.png)
+![Table Image] (images/sc1.png)
 
-## Demo - How can you use my Music Player?
+## Demo - How you can use my Music Player?
 
 ### 1. Install all the dependencies:
 - <a target="_blank" href="https://www.python.org/ftp/python/3.11.1/python-3.11.1-amd64.exe">Install Python</a>
 
 - <a target="_blank" href="https://docs.python.org/3/library/tkinter.html">Tkinter</a>
     ```
-    pip install tkinter
+    Pip install tkinter
     ```
 - <a target="_blank" href="https://docs.python.org/3/library/tkinter.html">Pygame</a>
     ```
-    pip install pygame
+    Pip install pygame
     ```
-- <a target="_blank" href="https://pypi.org/project/stagger/">stagger</a>
+- <a target="_blank" href="https://pypi.org/project/stagger/">Stagger</a>
     ```
-    pip install stagger
+    Pip install stagger
     ```
 - <a target="_blank" href="https://pillow.readthedocs.io/en/stable/">Pillow</a>
     ```
-    pip install pillow
+    Pip install pillow
     ```
-Note: If there are any other types of package installation needed install them also.
+Note: If there are any other types of package installation needed install then them also.
 
 ### 2. Run `Music_player.py` file
 - In command line
+
     ```
-    python Music_player.py
+    Python Music_player.py
```

Comment:
Instead of writing "Python" , "python" will be preferable in command line interface